src/Analyzers/CSharp/CodeFixes/UseCollectionExpression/CSharpUseCollectionExpressionForEmptyCodeFixProvider.cs

```diff
@@ -39,7 +39,7 @@ protected override bool IncludeDiagnosticDuringFixAll(Diagnostic diagnostic)
 
     public override Task RegisterCodeFixesAsync(CodeFixContext context)
     {
-        RegisterCodeFix(context, CSharpCodeFixesResources.Use_collection_expression, nameof(CSharpCodeFixesResources.Use_collection_expression));
+        RegisterCodeFix(context, CSharpCodeFixesResources.Use_collection_expression, IDEDiagnosticIds.UseCollectionExpressionForEmptyDiagnosticId);
```

Comment:
we want different equivalence keys so that we don't wrong the wrong fixers on the wrong diagnostics.